### DISCUSSION TITLE
LF5 My Open Math

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#380] [General]
Author: @AbbyANoble 
Reviewer: @tdegeorge 

Types of questions:

1-7: interpreting parameters, no calculations
8-10: writing equation only
11-16: write equation, and then find a y given an x
17-18: write equation, find a y given x, find an x given y
19-20: more things going on!

**Comment by tdegeorge** [comment]
Looks good to me! :)
